### PR TITLE
Update the JSS trial documentation to Lighthouse Fitness running on Sitecore 10.0.1

### PR DESCRIPTION
**Pull request by jflheureux**
Update the JSS trial documentation to Lighthouse Fitness running on Sitecore 10.0.1

## Motivation
HabitatHome Fitness is not maintained anymore. It was renamed to Lighthouse Fitness and now runs on Sitecore 10.0.1. The JSS trial was due for an update.

## How Has This Been Tested?
Tested all the modified pages of the documentation website locally using `jss start`

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] Documentation update (non-breaking change; modified files are limited to the `/docs` directory)

## Checklist:
- [x] I have read the Contributing guide.
- [x] My code follows the code style of this project.
- [x] My code/comments/docs fully adhere to the Code of Conduct.
- [ ] My change is a code change and it requires an update to the documentation.
- [ ] My change is a documentation change and it requires an update to the navigation.

### CODE REVIEW

**Comment by scottmulligan** [comment]
[Wait for confirmation from @jeanfrancoislarente on PR approval]

This updated documentation for the JSS Dev Trial should coincide with the release of  an updated version.

I have completed a QA review of the documentation changes in this PR and have confirmed they are ready to go with the updated version of the JSS Dev Trial.

File: docs/data/routes/connected-demo/explore-sitecore/experience-editor/en.md [hunk 1]
```diff
@@ -5,39 +5,31 @@ title: Experience Editor
...
-    </div>
-    <div class="col-md-6"> 
-      <p>Enter the Experience Editor from the Launchpad.</p>
-      <p>The Experience Editor is a WYSIWYG (What You See Is What You Get) editor that allows you to easily make changes to items directly on the page. You can edit all the items that are visible on the page — text, graphics, logos, links, and so on. In the Experience Editor, you can edit the fields of the current item and the fields of any items rendered on the page. For instance, if the menu on the page consists of titles of the product items, you can edit the titles without navigating to the product items themselves.</p>
-    </div>
-  </div>
-<p>
+The Experience Editor is a WYSIWYG (What You See Is What You Get) editor that allows you to easily make changes to items directly on the page. You can edit all the items that are visible on the page — text, graphics, logos, links, and so on. In the Experience Editor, you can edit the fields of the current item and the fields of any items rendered on the page. For instance, if the menu on the page consists of titles of the product items, you can edit the titles without navigating to the product items themselves.
```
Comment (anastasiya29) [inline docs/data/routes/connected-demo/explore-sitecore/experience-editor/en.md:9]: The markup was in there to render the image and text side by side, as 2 columns. IIRC, this way looked better or made it easier to understand what text goes with the image.

I'm not sure if that's still needed, but wanted to provide the context.

Comment (jeanfrancoislarente) [reply docs/data/routes/connected-demo/explore-sitecore/experience-editor/en.md:9]: Thanks for the context @anastasiya29! @jflheureux  will review your comment

Comment (jflheureux) [reply docs/data/routes/connected-demo/explore-sitecore/experience-editor/en.md:9]: I removed most of these side-by-side images and text in favor of numbered lists with images correctly aligned under each list item. Most of these side-by-side image/text were for instructions to follow in a specific order.

For this one in particular, the Launchpad "Experience Editor" button that was previously instructed to click on is not loading the correct site anymore so I had to change the instructions to go in the Content Editor first.

**Review by anastasiya29 — APPROVED** [review]

**Pull request merged**